### ISSUE TITLE
API: Document how to deploy/configure API

### ISSUE THREAD

**Issue by LakshmiLavanyaKasturi** [hirosystems/docs#322]
Per conversation in the API weekly meeting on 3-9, @CharlieC3, Please add your thoughts to improve deployment documentation.

**Comment by sabbyanandan** [comment]
@CharlieC3: With your new blog and the Hiro Archive docs update, do we still have any gaps? Or, we can close this as complete?

**Comment by CharlieC3** [comment]
@sabbyanandan This should remain open; the Archive docs update was only a small part and mostly pertains to overall API maintenance and doesn't really touch on how to deploy/configure the API.

**Comment by LakshmiLavanyaKasturi** [comment]
@CharlieC3 Could you help with the steps to configure and deploy API's?

**Comment by CharlieC3** [comment]
@LakshmiLavanyaKasturi Sorry I haven't forgotten about this, I just haven't had a chance to work on it yet.

**Comment by LakshmiLavanyaKasturi** [comment]
To address this issue: 

1. Any ETA on hosting a Bitcoin indexer these days? Once available, we can move off Blockstream entirely (which had some downtime last week...causing other headaches). Reference: https://hiropbc.slack.com/archives/C03UFUA5Q65/p1680012246276749 
2. Document steps to deploy and configure APIs. @CharlieC3 can help with more info.

**Comment by ryanwaits** [comment]
sounds like we are de-prioritizing content around spinning up nodes/apis atm

closing for now

**Comment by CharlieC3** [comment]
@ryanwaits I still feel like this is an important issue that should get addressed at some point. Can we leave it open?

**Comment by ryanwaits** [comment]
@CharlieC3 yeah thats fine with me, there were a few other issues that felt pretty similar that I've also been closing, so I'm fine leaving this one open

**Comment by CharlieC3** [comment]
@ryanwaits Are you able to own this and work with @zone117x on the updated details for deploying the API?

**Comment by ryanwaits** [comment]
> @ryanwaits Are you able to own this and work with @zone117x on the updated details for deploying the API?

i can do that
